AppCenterAnalytics/AppCenterAnalyticsTests/MSAnalyticsTests.m

```diff
@@ -562,6 +562,95 @@ - (void)testTrackEventWithTypedProperties {
   XCTAssertEqual([expectedProperties.properties count], 0);
 }
 
+- (void)testTrackEventWithNormalPersistenceFlag {
+
+  // If
+  __block NSString *type;
+  __block NSString *name;
+  NSString *expectedName = @"gotACoffee";
+  id channelUnitMock = OCMProtocolMock(@protocol(MSChannelUnitProtocol));
+  id channelGroupMock = OCMProtocolMock(@protocol(MSChannelGroupProtocol));
+  OCMStub([channelGroupMock addChannelUnitWithConfiguration:OCMOCK_ANY]).andReturn(channelUnitMock);
+  OCMStub([channelUnitMock enqueueItem:[OCMArg isKindOfClass:[MSEventLog class]] flags:MSFlagsPersistenceNormal])
+      .andDo(^(NSInvocation *invocation) {
+        MSEventLog *log;
+        [invocation getArgument:&log atIndex:2];
+        type = log.type;
+        name = log.name;
+      });
+  [MSAppCenter configureWithAppSecret:kMSTestAppSecret];
+  [[MSAnalytics sharedInstance] startWithChannelGroup:channelGroupMock
+                                            appSecret:kMSTestAppSecret
+                              transmissionTargetToken:nil
+                                      fromApplication:YES];
+
+  // When
+  [MSAnalytics trackEvent:expectedName withTypedProperties:nil flags:MSFlagsPersistenceNormal];
+
+  // Then
+  assertThat(type, is(kMSTypeEvent));
+  assertThat(name, is(expectedName));
```

Comment:
it would be better to stub with any flag and assertThat flag is normal by capturing it, it took me some time to realize why the test was testing the flag.

Reply:
(Multiple place comment)

Reply:
I only changed unit tests that I created. The rest tests are not required to check flags.